### DISCUSSION TITLE
game launchers

### DISCUSSION THREAD

**Discussion by Gedankenn** [prasanthrangan/hyprdots#192] [Q&A]
I saw in the readme of the git repo some screenshots of a game launcher, if someone could elaborate a little more about wich app that is, i found that incredible beautifull <3. thanks in advance.

**Comment by T-Crypt** [comment]
It pulls your Steam library -- You could set a keybind for the gamelauncher.sh script in .config/hypr/scripts/gamelauncher.sh

**Reply by Gedankenn** [reply]
nice. thanks

**Reply by prasanthrangan** [reply]
also please don't forget to give it a parameter to run -
```
./gamelauncher.sh <n> # where n is style [1-4] 
```

**Reply by Braeulias** [reply]
my steam library isnt found, how can i fix that?

**Comment by PassertlonG** [comment]
nothing but dontkillSteam.sh
